rdt-open-reader/src/test/java/io/ona/rdt/robolectric/util/UtilsTest.java

```diff
@@ -275,6 +279,23 @@ public void testExceptionIsCaughtFromUserAuthorizationTask() throws Exception {
         ReflectionHelpers.setField(userAuthorizationVerificationTask, "syncUtils", syncUtils);
         userAuthorizationVerificationTask.execute();
 
+        ReflectionHelpers.setStaticField(Utils.UserAuthorizationVerificationTask.class, "INSTANCE", null);
+        RDTApplication.getInstance().clearCurrActivityReference(patientRegisterActivity);
+    }
+
+    @Test
+    public void testLogoutShouldNotCallWhenCurrentActivityIsNotPatientRegisterActivity() throws Exception {
```

Comment:
rename to `testUVerifyUserAuthorizationShouldNotLogoutUserIfNotOnRegistrationPage`

Reply:
done